src/errors/test-run/templates.js

```diff
@@ -306,5 +306,14 @@ export default {
 
         Error details:
         ${escapeHtml(err.errMsg)}
+    `),
+
+    [TEST_RUN_ERRORS.executeAsyncExpressionError]: err => markup(err, `
+        An unhandled error occurred in a step with custom JS code:
```

Comment:
BTW we use  `action` instead of `step`. Of course it's minor but still let's stick to our terminology.

Reply:
@VasilyStrelyaev please take a look on this

Reply:
This type of step, (BTW, my new favorite name is `Run TestCafe Script`), actually illustrates that steps and actions are different things.
It's a step that can contain multiple actions inside.

We won't rename `Actions` panel to `Steps` panel because of that, but still 'step' is a more generic name for TestCafe Studio test's elementary parts, while 'actions' are more about what the TestCafe engine does in these steps.

Yes, now we say 'actions' almost everywhere. However, for instance, in hooks I would say 'move/copy steps to hook' to emphasize that TestCafe Studio entities are copied regardless of what they do (that's a vague explanation, but still)...

Reply:
So, I would leave this message as is

Reply:
@VasilyStrelyaev Thank you for such a comprehensive explanation